help/zh_CN/commands.yml

```diff
@@ -1,138 +1,148 @@
             |
                 {l}{6}/{r}camera {8}step{r} {7}[x] [y] [z]{r}
-                
+
                 {r}根据绝对的数值或增减幅度设定摄像机位置。你可以通过指定数字设定绝对的摄像机位置 {6}/camera {8}step {7}0 4 0{r}. 或者你可以通过{7}+{r} 或 {7}-{r} 标志来增加或减少位置数值 {6}/camera {8}step {7}+0 +0.1 +0{r}。很像 {6}/tp{r} 命令。
 
         fixture:
-            add: 
+            add:
                 |
                     {l}{6}/{r}camera {8}add{r} {7}<idle|path|follow|look|circular> <duration>{r}
-                    
+
                     {r}在相机配置的最后添加一个指定的类型和 {7}<duration>{r}（时长）的相机关键点。
-            edit: 
+            edit:
                 |
                     {l}{6}/{r}camera {8}edit{r} {7}<index> <values...>{r}
-                    
+
                     {r}在目前的相机配置里设定索引号 {7}<index>{r} 关键点的参数。
-            remove: 
+            remove:
                 |
                     {l}{6}/{r}camera {8}remove{r} {7}<index>{r}
-                    
+
                     {r}移除索引号 {7}<index>{r} 的相机关键点。
-                    
-            duration: 
+
+            duration:
                 |
                     {l}{6}/{r}camera {8}duration{r} {7}[index] [duration]{r}
-                    
+
                     {r}可以用来设置，或者获取整个相机配置或一个相机关键点的时长。如果所有的参数按照预设，命令会设置{7}[duration]{r} (tick单位)时长到索引号 {7}[index]{r} 的相机关键点。增减符号可以用在 {7}[duration]{r} 参数。
-                
-            move: 
+
+            move:
                 |
                     {l}{6}/{r}camera {8}move{r} {7}<from> <to>{r}
-                    
+
                     {r}在相机配置里，将关键点索引号 {7}<from>{r} 移动到索引号 {7}<to>{r}。
-            
-        path: 
+
+        path:
             help: "{r}{o}相机路径次级命令{r}。提供管理相机路径和关键点的次级命令。"
-            
+
             add:
                 |
                     {l}{6}/{r}camera path {8}add{r} {7}<index> [before_point]{r}
-                    
+
                     {r}在索引号 {7}<index>{r} 的后面添加一个路径点，或者在指定 {7}[before_point]{r} 编号的前面(如果指定了)。
-            
+
             edit:
                 |
                     {l}{6}/{r}camera path {8}edit{r} {7}<index> <point_index>{r}
-                    
+
                     {r}在路径索引号 {7}<index>{r} 的路径里编辑路径点索引号 {7}<point_index>{r} 的位置。
-            
+
             goto:
                 |
                     {l}{6}/{r}camera path {8}goto{r} {7}<index> <point_index>{r}
-                    
+
                     {r}在路径索引号 {7}<index>{r} 的路径里前往路径点索引号 {7}<point_index>{r} 的位置。
-            
+
             move:
                 |
                     {l}{6}/{r}camera path {8}move{r} {7}<index> <from> <to>{r}
-                    
+
                     {r}在路径索引号 {7}<index>{r} 的路径里将路径点索引号 {7}<from>{r} 移动到路径点索引号 {7}<to>{r}。
-            
+
             remove:
                 |
                     {l}{6}/{r}camera path {8}remove{r} {7}<index> <point_index>{r}
-                    
-                    {r}在路径索引号 {7}<index>{r} 的路径里移除路径点索引号 {7}<point_index){r}。
\ No newline at end of file
+
+                    {r}在路径索引号 {7}<index>{r} 的路径里移除路径点索引号 {7}<point_index){r}。
```

Comment:
Hm, this line is a bit weird. I think it should be `<point_index>` instead of `<point_index)`.